src/test/java/org/apache/commons/io/IOUtilsTest.java

```diff
@@ -1716,4 +1718,12 @@ public void testWriteLittleString() throws IOException {
         }
     }
 
+    @Test
+    public void testByteArrayWithNegativeSize() {
+        int size = -1;
+        assertThrows(NegativeArraySizeException.class,() -> {
```

Comment:
You can just say `assertThrows(NegativeArraySizeException.class,() -> IOUtils.byteArray(-1));` but I am not even sure we should have this test since it does not improve our code coverage, it just test what the JVM does.

Reply:
Hi garydgregory,
    from your comments, I checked the code and fixed again, you can review it again.
Thanks.